chromium_src/chrome/browser/prefs/browser_prefs.cc

```diff
@@ -265,6 +266,7 @@ void MigrateObsoleteLocalStatePrefs(PrefService* local_state) {
 #endif
 
   brave_search_conversion::p3a::MigrateObsoleteLocalStatePrefs(local_state);
+  p3a::StarRandomnessMeta::MigrateObsoleteLocalStatePrefs(local_state);
```

Comment:
These likely need to be registered in iOS as well: https://github.com/brave/brave-core/blob/master/chromium_src/ios/chrome/browser/shared/model/prefs/browser_prefs.mm

Reply:
is there a local state migration function for iOS? i see a profile one, but not a local state one

Reply:
We may not have had the need for it yet? Will possibly need to add the patch for it